### ISSUE TITLE
Analyser should not segfault with a null model pointer.

### ISSUE THREAD

**Issue by kerimoyle** [cellml/libcellml#713]
Currently passing a `nullptr` causes a segfault in the `Analyser`.  It should test for this and exit gracefully if found ... @agarny ?

```
TEST(Analyser, nullModel){
    auto analyser = libcellml::Analyser::create();
    analyser->analyseModel(nullptr); // << SEGFAULTS HERE
    EXPECT_TRUE(true);
}
```

**Comment by agarny** [comment]
Ok, I am dealing with this in PR #631 since it really ought to have been done there. Thanks.

**Comment by agarny** [comment]
Reopening since this should only be closed once I have implemented in PR #631.

**Comment by agarny** [comment]
Closing as a result of commit 3008b3b.
